manifests/integreatly-unifiedpush/unifiedpush-operator-0.3.0/unifiedpush-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -237,7 +242,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: unifiedpush-operator
-                image: quay.io/aerogear/unifiedpush-operator:0.3.0
+                image: quay.io/dimitra/unifiedpush-operator:f1d88ec
```

Comment:
Pointing to my own quay until a new version of the ups operator is released
/cc @grdryn

Reply:
Will create a new manifest at that point as well